### DISCUSSION TITLE
tiktok.com - banner download apps

### DISCUSSION THREAD

**Discussion by 3xploiton3** [realodix/AdBlockID#1108] [Personal Filter]
**Jenis**

**Sample URL**
https://www.tiktok.com/id-ID/?is_copy_url=1&is_from_webapp=1

**Penjelasan Lebih Lanjut**

![Screenshot_20220129-204949144 (1)](https://user-images.githubusercontent.com/19517680/151663576-49862b75-f7dd-4d50-8df0-e4707631c442.jpg)

**Comment by realodix** [comment]
**Bahasa Indonesia**
`tiktok.com##.footer-guide:has-text(aplikasi):has(button)`

**International**
`tiktok.com##.footer-guide:has-text(on the app):has(button)`
